src/Header.tsx

```diff
@@ -0,0 +1,82 @@
+import cn from 'classnames';
+import { useContext, useEffect, useRef } from 'react';
+import { patchTodo } from './api/todos';
+import { ContextTodos } from './TodoContext';
+
+export const Header = () => {
+  const {
+    setErrMessage,
+    setIsLoading,
+    setTodos,
+    isLoading,
+    handleSubmit,
+    newTitle,
+    setNewTitle,
+    todos,
+  } = useContext(ContextTodos);
+  const isAllActive = todos.every(todo => todo.completed);
+
+  const selectInputTitle = useRef<HTMLInputElement>(null);
+
+  //focus on input for creating new title
+  useEffect(() => {
+    if (selectInputTitle.current) {
+      selectInputTitle.current.focus();
+    }
+  }, [isLoading]);
```

Comment:
you have focus in your app.tsx remove 1 of them

Reply:
this is for useRef and both are for diffrent inputs, i found better place for this in header ;)